### DISCUSSION TITLE
Chummer on Mac with Apple CPU (M1)

### DISCUSSION THREAD

**Discussion by blacksemid** [chummer5a/chummer5a#4523] [Q&A] [answered]
Hello, 
is it possible to start Chummer on a new Mac with M1 CPU?
Maybe with Wine, Mono or Docker?
has someone tested this and can tell me whether this works?

**Comment by DelnarErsike** [comment, marked as answer]
I believe your best option would be to use CodeWeavers' CrossOver, at least until their M1 Wine compatibility patches are ported to Wine proper.
